### DISCUSSION TITLE
感觉要做大做强还是得弄个服务器

### DISCUSSION THREAD

**Discussion by YujingtaiAqing** [share121/inter-knot#5329] [灌水]
感觉要做大做强还是得弄个服务器，现在这个github跳来跳去有点门槛了，顺便晒晒我刚到的停云🥰
![IMG_20240921_155947](https://github.com/user-attachments/assets/73a46083-7601-4559-9340-f986837f03dd)

**Comment by Fluorine-z** [comment]
这是不是官方价二百在淘宝被炒到二百五六的那个

**Reply by HoshinoHanashi** [reply]
这个手办所有都在预定，今天刚到，谁家能炒到二百五六，都是预定199啊<img src="https://bbs-static.miyoushe.com/static/2023/12/22/b7e1e6889431e530705df3b830422614_3687690709658525720.png" alt="符华-推理" width="75" >

**Reply by YujingtaiAqing** [reply]
阿哲，米家手办居然还有必要炒吗？预定了后面慢慢都会有的，真的会有人加价买吗
